cost-analyzer/values.yaml

```diff
@@ -22,6 +22,23 @@ global:
     proxy: true # If true, the kubecost frontend will route to your grafana through its service endpoint
 #    fqdn: cost-analyzer-grafana.default.svc
 
+  # Google Cloud Managed Service for Prometheus
+  gmp:
+  # Remember to set up these parameters when install the Kubecost Helm chart with `global.gmp.enabled=true` if you want to use GMP self-deployed collection (Recommended) to ultilize Kubecost scrape configs.
```

Comment:
@linhlam-kc Is it safe to remove this comment here? I believe it's self evident that these configs are required if you are planning to configure Kubecost with GMP?

Reply:
I think we should let it there with the assumption that the new users may not be familiar with what GMP stand for